src/libraries/System.Formats.Cbor/src/System/Formats/Cbor/Writer/CborWriter.Map.cs

```diff
@@ -14,36 +14,23 @@ public partial class CborWriter
-        /// </exception>
-        /// <exception cref="InvalidOperationException">
-        ///   Writing a new value exceeds the definite length of the parent data item. -or-
-        ///   The major type of the encoded value is not permitted in the parent data item. -or-
-        ///   The written data is not accepted under the current conformance mode.
-        /// </exception>
+        /// <summary>Writes the start of a definite or indefinite-length map (major type 5).</summary>
+        /// <param name="definiteLength">The length of the definite-length map, or <see langword="null" /> for an indefinite-length map.</param>
+        /// <exception cref="ArgumentOutOfRangeException">The <paramref name="definiteLength" /> parameter cannot be negative.</exception>
+        /// <exception cref="InvalidOperationException">Writing a new value exceeds the definite length of the parent data item.
+        /// -or-
+        /// The major type of the encoded value is not permitted in the parent data item.
+        /// -or-
+        /// The written data is not accepted under the current conformance mode.</exception>
         /// <remarks>
-        ///   <para>
-        ///     In canonical conformance modes, the writer will reject indefinite-length writes unless
-        ///     the <see cref="ConvertIndefiniteLengthEncodings"/> flag is enabled.
-        ///   </para>
-        ///   <para>
-        ///     Map contents are written as if arrays twice the length of the map's declared size.
-        ///     For instance, a map of size <c>1</c> containing a key of type int with a value of type string
-        ///     must be written by successive calls to <see cref="WriteInt32(int)"/> and <see cref="WriteTextString(ReadOnlySpan{char})"/>.
-        ///     It is up to the caller to keep track of whether the next call is a key or a value.
-        ///   </para>
-        ///   <para>
-        ///     Fundamentally, this is a technical restriction stemming from the fact that CBOR allows keys of any type,
-        ///     for instance a map can contain keys that are maps themselves.
-        ///   </para>
+        /// In canonical conformance modes, the writer will reject indefinite-length writes unless
+        /// the <see cref="ConvertIndefiniteLengthEncodings" /> flag is enabled.
+        /// Map contents are written as if arrays twice the length of the map's declared size.
+        /// For instance, a map of size `1` containing a key of type int with a value of type string must be written
+        /// by successive calls to <see cref="WriteInt32(int)" /> and <see cref="WriteTextString(System.ReadOnlySpan{char})" />.
+        /// It is up to the caller to keep track of whether the next call is a key or a value.
+        /// Fundamentally, this is a technical restriction stemming from the fact that CBOR allows keys of any type,
+        /// for instance a map can contain keys that are maps themselves.
```

Comment:
This `int` word was not wrapped by backticks so it wasn't detected by the tool (not a bug):

```suggestion
        /// In canonical conformance modes, the writer will reject indefinite-length writes unless
        /// the <see cref="ConvertIndefiniteLengthEncodings" /> flag is enabled.
        /// Map contents are written as if arrays twice the length of the map's declared size.
        /// For instance, a map of size 1 containing a key of type <see cref="System.Int32" /> with a value of type string must be written
        /// by successive calls to <see cref="WriteInt32(int)" /> and <see cref="WriteTextString(System.ReadOnlySpan{char})" />.
        /// It is up to the caller to keep track of whether the next call is a key or a value.
        /// Fundamentally, this is a technical restriction stemming from the fact that CBOR allows keys of any type,
        /// for instance a map can contain keys that are maps themselves.
```